frontend/src/utils/d3/graph.js

```diff
@@ -0,0 +1,224 @@
+import * as d3 from 'd3';
+
+const inverseEdgeType = {
+  source: 'target',
+  target: 'source',
+};
+
+/**
+ * Get angle of line connecting points, in radians
+ * @param {int} x1 x of first point
+ * @param {int} y1 y of first point
+ * @param {int} x2 x of second point
+ * @param {int} y2 y of second point
+ */
+function getAngle(x1, y1, x2, y2) {
+  const delta_x = x2 - x1;
+  const delta_y = y2 - y1;
+  const theta_radians = Math.atan2(delta_y, delta_x);
+  return theta_radians;
+}
+
+/**
+ * Get the X position of a point on the circumference of a circle
+ * @param {int} cx x of center of circle
+ * @param {int} cy y of center of circle
+ * @param {int} x x of point outside circle
+ * @param {int} y y of point outside circle
+ * @param {int} r radius of circle
+ * @returns x of point on circumference of circle
+ */
+function getXonCircumference(cx, cy, x, y, r) {
```

Comment:
```suggestion
function getXOnCircumference(cx, cy, x, y, r) {
```

Reply:
Removed this function in a224a58cecd993df9788e94249d0122dfa40abdc